components/data/include/data.h

```diff
@@ -5,4 +5,7 @@ typedef struct {
     int rssi;
 } bluetooth_frame;
```

Comment:
We might want to talk about changing the name of our custom data type to like, `csha_bt_packet` just to avoid confusing it with literally a [bluetooth frame](https://www.tutorialspoint.com/the-bluetooth-frame-structure)

Reply:
`csha` stands for CSHAnalytics (I'm gonna make that a legacy thing :P)